seq_logger.js

```diff
@@ -284,78 +275,47 @@ class SeqLogger {
 
         return new Promise((resolve, reject) => {
             const sendRequest = (batch, bytes) => {
+                const controller = new AbortController()
```

Comment:
To support node 14 we must handle missing `AbortController`.

Something like:

```js
const controller = typeof(AbortController) === 'undefined' 
                    ? () => ({ abort: () => {} }) 
                    : new AbortController();
```

Reply:
I think better to use the [abort-controller](https://www.npmjs.com/package/abort-controller) since we can provide the same functionality.